client/app/bundles/server-components/components/ServerInfo.jsx

```
@@ -0,0 +1,58 @@
+    nodeVersion: 'Node.js',
+    uptime: 'Uptime (hrs)',
+    totalMemory: 'Total RAM (GB)',
+    freeMemory: 'Free RAM (GB)',
+    cpus: 'CPU Cores',
+    hostname: 'Hostname',
+  };
+
+  return (
+    <div className="bg-gradient-to-br from-emerald-50 to-teal-50 border border-emerald-200 rounded-xl p-6">
+      <p className="text-xs text-emerald-600 mb-4 font-medium">
+        This data comes from the Node.js <code className="bg-emerald-100 px-1 rounded">os</code> module
```

ESLint

[error] Insert `·—·it`

(prettier/prettier)